packages/atomic/.storybook/default-story-shared.tsx

```diff
@@ -90,6 +90,12 @@ export function renderShadowPartsToStyleString(
   return styleElement.outerHTML;
 }
 
+export function renderAdditionalMarkup(additionalMarkup: () => TemplateResult) {
```

Comment:
After reading a bit more on lit-html documentation, this is actually how the `TemplateResult` API is meant to be used, and trying to obtain a string from a complex template that embeds multiple HTML function was getting pretty complex.